### DISCUSSION TITLE
Game crashes on startup

### DISCUSSION THREAD

**Discussion by Craanky** [ppy/osu#24888] [Q&A] [answered]
My game just crashes on startup. I use Linux and the flathub version of Lazer
[database.log](https://github.com/ppy/osu/files/12705885/database.log)
[input.log](https://github.com/ppy/osu/files/12705886/input.log)
[network.log](https://github.com/ppy/osu/files/12705887/network.log)
[performance.log](https://github.com/ppy/osu/files/12705888/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12705889/runtime.log)

**Comment by peppy** [comment]
The game is not reporting an error, so you're going to have to search elsewhere on your system for error logs.

**Comment by Craanky** [comment, marked as answer]
ok, i installed the appimage and it worked, can we fix the flathub somehow?

**Reply by peppy** [reply]
for sure! if you know the issue you can actually submit a fix as it's open source.

https://github.com/flathub/sh.ppy.osu
